doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
 
-1. Add an additional replica to the `quickstart` cluster:
+1. Switch to the `compute_qck` cluster and query the materialized view:
 
     ```sql
-    CREATE CLUSTER REPLICA quickstart.bigger SIZE = 'xsmall';
+    SET CLUSTER = compute_qck;
     ```
 
-1. To simulate a failure, drop the `small_replica`:
+    ```sql
+    SELECT * FROM num_bids;
+    ```
+    <p></p>
+
+    ```nofmt
+            item        | number_of_bids
+    --------------------+----------------
+     Custom Art         |          10634
+     Gift Basket        |          11266
+     City Bar Crawl     |          10292
+     Best Pizza in Town |          10498
+     Signed Memorabilia |          10801
+    (5 rows)
+
+    Time: 790.384 ms
+    ```
+
+    As you can see, although the materialized view was created in the `default` cluster, its results are available from other clusters as well because they are in shared, durable storage.
+
+1.  If retrieving a materialized view's results from storage is too slow, you can create an index on the materialized view as well:
 
     ```sql
-    DROP CLUSTER REPLICA quickstart.small_replica;
+    CREATE INDEX num_bids_idx ON num_bids (item);
     ```
 
-    If you switch to the terminal window running the `SUBSCRIBE` command, you'll see that results are still being pushed out to the client!
+    ```sql
+    SELECT * FROM num_bids;
+    ```
+    <p></p>
 
-## What's next?
+    ```nofmt
+            item        | number_of_bids
+    --------------------+----------------
+     Custom Art         |          14373
+     Gift Basket        |          15271
+     City Bar Crawl     |          14294
+     Best Pizza in Town |          14606
+     Signed Memorabilia |          14843
+    (5 rows)
+
+    Time: 32.064 ms
+    ```
 
-That's it! You just created your first transformations on streaming data using common SQL patterns, and tried to break Materialize! We encourage you to continue exploring the sample Kafka source, or try one of the use-case specific quickstarts. If you’re done for the time being, remember to clean up the environment:
+    Now that the materialzed view's results are served from memory, latency is low again.
 
-```sql
-DROP SCHEMA qck CASCADE;
+## Step 5. Survive failures
+
+Earlier, when you created your clusters, you gave each cluster one [replica](/overview/key-concepts/#cluster-replicas), that is, one physical resource. For the `ingest_qck` cluster, that's the max number of replicas allowed, as clusters for sources can have only one replica. For the `compute_qck` cluster, however, you can increase the number of replicas for greater tolerance to replica failure.
+
+Each replica in a non-source cluster is a logical clone, doing the same computation and holding the same results in memory. This design provides Materialize with active replication, and so long as one replica is still reachable, the cluster continues making progress.
+
+Let's see this in action.
```

Comment:
```suggestion
In this step, you'll add a replica and simulate a cluster replica failure.
```

Reply:
Transition feels odd without this. We can revisit.